src/Config/Parser/AnnotationParser.php

```diff
@@ -871,22 +872,23 @@ private static function guessType(string $namespace, array $annotations): string
             $target = self::fullyQualifiedClassName($associationAnnotation->targetEntity, $namespace);
             $type = self::resolveTypeFromClass($target, ['type']);
 
-            if ($type) {
-                $isMultiple = $associationAnnotations[\get_class($associationAnnotation)];
-                if ($isMultiple) {
-                    return \sprintf('[%s]!', $type);
-                } else {
-                    $isNullable = false;
-                    $joinColumn = self::getFirstAnnotationMatching($annotations, JoinColumn::class);
-                    if ($joinColumn) {
-                        $isNullable = $joinColumn->nullable;
-                    }
+            if (!$type) {
+              throw new \RuntimeException(\sprintf('Unable to auto-guess GraphQL type from Doctrine target class "%s" (check if the target class is a GraphQL type itself (with a @GQL\Type annotation).', $target));
```

Comment:
```suggestion
                throw new \RuntimeException(\sprintf('Unable to auto-guess GraphQL type from Doctrine target class "%s" (check if the target class is a GraphQL type itself (with a @GQL\Type annotation).', $target));
```

Fix CS: add 2 more spaces before `throw`